opentracing-api/pom.xml

```diff
@@ -25,8 +25,8 @@
         <version>0.1.0-SNAPSHOT</version>
     </parent>
 
-    <artifactId>opentracing</artifactId>
-    <name>OpenTracing</name>
+    <artifactId>opentracing-api</artifactId>
```

Comment:
What I've got here is a common way of naming the artifacts.

Using the `-api` suffix here means the jar file is named opentracing-api.jar.
Users will also want to define the api as a compile dependency and then the chosen implementation, `opentracing-brave`, as a provided dependency. That the api suffix is explicit makes it obvious to why the api artifact is a visibly exposed dependency while the implementation and bridge are not.